frontend/src/components/GeyserUnstakeView.tsx

```diff
@@ -19,33 +19,35 @@ export const GeyserUnstakeView: React.FC<Props> = () => {
   const [amount, setAmount] = useState<string>('')
   const [receipt, setReceipt] = useState<TransactionReceipt>()
   const [parsedAmount, setParsedAmount] = useState<BigNumber>(BigNumber.from('0'))
-  const { selectedGeyser, stakingTokenDecimals, stakingTokenSymbol } = useContext(GeyserContext)
+  const { selectedGeyser, stakingTokenInfo } = useContext(GeyserContext)
+  const { decimals: stakingTokenDecimals, symbol: stakingTokenSymbol } = stakingTokenInfo
   const { signer } = useContext(Web3Context)
   const { selectedVault, currentLock } = useContext(VaultContext)
   const { walletAmount, refreshWalletAmount } = useContext(WalletContext)
 
   const userStake = BigNumber.from(amountOrZero(currentLock?.amount))
 
-  useEffect(() => {
-    refresh()
-  }, [receipt])
-
   const refresh = () => {
     setAmount('')
     setParsedAmount(BigNumber.from('0'))
     refreshWalletAmount()
   }
 
+  useEffect(() => {
+    refresh()
+  }, [receipt])
+
   const handleUnstake = async () => {
     if (selectedGeyser && selectedVault && signer) {
       const geyserAddress = selectedGeyser.id
       const vaultAddress = selectedVault.id
-      const tx = await unstakeWithdraw(geyserAddress, vaultAddress, parsedAmount, signer as Wallet)
-      setReceipt(await tx.wait())
+      const txs: [TransactionResponse, TransactionResponse] = await unstakeWithdraw(geyserAddress, vaultAddress, parsedAmount, signer as Wallet)
+      const [withdrawStakingTokenTx] = txs
```

Comment:
since we just need the first value, we can destructure it directly in the await
`const [withdrawStakingTokenTx] = await unstakeWithdraw(geyserAddress, vaultAddress, parsedAmount, signer as Wallet)`